charts/logzio-telemetry/templates/_config.tpl

```diff
@@ -1,39 +1,13 @@
-{{/*
-Default memory limiter configuration for OpenTelemetry Collector based on k8s resource limits.
-*/}}
-{{- define "opentelemetry-collector.memoryLimiter" -}}
-# check_interval is the time between measurements of memory usage.
-check_interval: 5s
 
-# By default limit_mib is set to 80% of ".Values.resources.limits.memory"
-limit_mib: {{ include "opentelemetry-collector.getMemLimitMib" .Values.resources.limits.memory }}
-
-# By default spike_limit_mib is set to 25% of ".Values.resources.limits.memory"
-spike_limit_mib: {{ include "opentelemetry-collector.getMemSpikeLimitMib" .Values.resources.limits.memory }}
-{{- end }}
 
 {{/*
 Merge user supplied top-level (not particular to standalone or agent) config into memory limiter config.
 */}}
 {{- define "opentelemetry-collector.baseConfig" -}}
 {{- $processorsConfig := get .Values.baseCollectorConfig "processors" }}
-{{- if not $processorsConfig.memory_limiter }}
-{{- $_ := set $processorsConfig "memory_limiter" (include "opentelemetry-collector.memoryLimiter" . | fromYaml) }}
-{{- end }}
 {{- .Values.baseCollectorConfig | toYaml }}
 {{- end }}
 
-{{/*
-Build config file for agent OpenTelemetry Collector
-*/}}
-{{- define "opentelemetry-collector.agentCollectorConfig" -}}
```

Comment:
Did we check that removing all the deprecated components and values does not break anything in our deployment?

Reply:
Ofcourse